src/utils/auth/auth.provider.tsx

```diff
@@ -84,6 +86,30 @@ export function AuthRequestsProvider({
     []
   );
 
+  const closeAuthPopup = useCallback((delay: number = 0) => {
```

Comment:
Small change for the embedded wallet already here, but it doesn't affect the functionality in this PR.